src/gps_helper.h

```diff
@@ -98,6 +98,13 @@ enum class GPSCallbackType {
 	setClock,
 };
 
+enum class GPSRestartType {
+	None,
+	Hot,
```

Comment:
Can you add some documentation what the exact meaning is for each type?

Reply:
Added descriptions from UBX documentation.